tests/css_trailing_comma/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -74,42 +74,48 @@ $breakpoint-map: (
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 $colors: hotpink deepskyblue firebrick;
 $list: (a);
-$list: (
-  "Helvetica",
-  "Arial",
-  sans-serif
-);
-$colors: (
-  "red",
-  "blue"
-);
+$list: ("Helvetica", "Arial", sans-serif);
```

Comment:
@j-f1 what do you think, should we print list as map? Original issue request this, but i am not sure

Reply:
Expected. Why https://github.com/prettier/prettier/pull/4487#issuecomment-389152753